src/core/BaseNode.js

```diff
@@ -245,5 +178,72 @@ export default Class(null, {
    * @method exit
    * @param {Tick} tick A tick instance.
    **/
-  exit: function(tick) {},
-});
+  exit(tick) {}
+};
+
+/**
+ * Node ID.
+ * @property {string} id
+ * @readonly
+ **/
+BaseNode.prototype.id = null;
```

Comment:
Yup, `id` could be dropped from this prototype. What about other properties?
For example: `name` and `title` are re-defined in `Wait.prototype`, but `Succeeder` only defines `name`.
Seems like some properties of prototypes are not required.
Now that I think about it, maybe it's better to keep other properties of BaseNode's prototype as they are. They serve a nice description for what can be used when extending each class.